docs/developing/legacy/index.rst

```diff
@@ -0,0 +1,8 @@
+====================
+Legacy documentation
+====================
+
+.. toctree::
+   :maxdepth: 2
+
+   pluginsv0/index
```

Comment:
Replace this page by pluginsv0/index, such that there is one less level of indirection.

Reply:
addressed here https://github.com/overhangio/tutor/compare/5f70a2e19267449ed9fe5b35a80a3daef6366b2d..7eb67a7705d82a0038855bc1003f95499c2b3470